src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig

```diff
@@ -78,12 +92,19 @@
     </tfoot>
 </table>
 
-<div id="sylius-cart-coupons" class="well well-sm" data-prototype="{{ form_row(form.promotionCoupons.vars.prototype)|e }}">
-    <div id="sylius-cart-coupons-collection">
-        {{ form_row(form.promotionCoupons, {'label_attr': {'class': 'col-lg-12'}}) }}
+<div class="col-md-12 well well-sm">
+    {{ form_label(form.promotionCoupons, '', { 'label_attr': {'class': 'col-md-12'} }) }}
+    <div id="sylius-cart-coupons" class="collection-container"
+         data-prototype="{{ _self.coupon_item(form.promotionCoupons.vars.prototype)|e }}">
+        {% for promotionCoupon in form.promotionCoupons %}
+            {{ _self.coupon_item(promotionCoupon) }}
+        {% endfor %}
     </div>
+
     <br>
-    <a href="#" class="btn btn-success btn-block" data-collection-button="add" data-prototype="sylius-cart-coupons" data-collection="sylius-cart-coupons-collection">
+    <a href="#" class="btn btn-success btn-block" data-collection-button="add"
```

Comment:
It is a collection, you should use the Form Collection Extension which provide default template and JS/